src/ui/handlers.py

```
@@ -37,20 +37,32 @@ def handle_gradio_input(
     # Call text processing logic first
-    response_text, updated_history, updated_history_for_gradio, updated_order, _ = process_order(
-        user_input_text=user_input,
-        current_session_history=session_history_state,
-        llm=llm,
-        rag_index=rag_index,
-        rag_documents=rag_documents,
-        rag_retriever=rag_retriever,
-        api_key=api_key
-    )
+    try:
+        response_text, updated_history, updated_history_for_gradio, updated_order, _ = process_order(
```

Ruff (0.13.1)

Unpacked variable `updated_history_for_gradio` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)